platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/platform/SnapshotEvent.java

```diff
@@ -16,14 +16,16 @@
 
 package com.swirlds.common.metrics.platform;
 
-import static com.swirlds.common.utility.CommonUtils.throwArgNull;
-
 import com.swirlds.common.platform.NodeId;
 import java.util.Collection;
+import java.util.Objects;
 
 public record SnapshotEvent(NodeId nodeId, Collection<Snapshot> snapshots) {
 
+    /**
+     * @throws NullPointerException in case {@code metric} parameter is {@code null}
```

Comment:
Copy/paste error? The variable checked is `snapshots` here...

Reply:
Fixed